pkg/storage/business_case.go

```diff
@@ -343,7 +343,9 @@ func (s *Store) UpdateBusinessCase(businessCase *models.BusinessCase) (*models.B
 			alternative_b_cons = :alternative_b_cons,
 			alternative_b_cost_savings = :alternative_b_cost_savings,
 			updated_at = :updated_at,
-		    status = :status
+		    status = :status,
+			initial_submitted_at = :submitted_at,
```

Comment:
Does this need to point to the new column instead? `initial_submitted_at` vs `submitted_at`?